config/environments/development.rb

```diff
@@ -44,11 +47,6 @@
   # Tell Active Support which deprecation messages to disallow.
   config.active_support.disallowed_deprecation_warnings = []
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  config.assets.debug = true
```

Comment:
[Auto generated change]
This setting is default in v7.0.x.
If we handle JavaScript's files using Sprockets, this setting is useful to debug the codes.
But we handle them using Webpacker so we don't need this setting because Webpacker generates source map for debugging.

https://github.com/rails/webpacker/blob/1cec8408d9c30e458c9f83b0c50ef53a255a4352/package/environments/development.js#L11-L12

ref: https://github.com/rails/rails/blob/7-0-stable/railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt
ref: https://guides.rubyonrails.org/asset_pipeline.html#turning-source-maps-on